umbraco/Umbraco-CMS
# v9: Don't load all translation files

Fixes https://github.com/umbraco/Umbraco-CMS/issues/11438

# Notes
- Added lazy loading to translation files so we don't load all of them when only using 1 language
- Fixed unit tests

I've posted an image here where you see everytime we load in a language we use more memory, here 21 files are loaded in: 
![image](https://user-images.githubusercontent.com/70372949/140294532-45c0a0df-08f4-4866-98ee-071cfef9bbd7.png)

In the new one, all those are gone (only 1 language is loaded) 
![image](https://user-images.githubusercontent.com/70372949/140297040-2d360495-b17c-4beb-a888-bbd0cd3f9997.png)


# How to test
- Run the umbraco installer and close the website afterwards
- Set a breakpoint on line 118 in LocalizedTextServiceFileSources like so:
![image](https://user-images.githubusercontent.com/70372949/140297367-5726e525-3a08-4109-88a0-43c527e544f2.png)
- Run Umbraco again, this should only load the languages you are using(Before this would load all languages)
- Go to user settings and change your language
- We should again only be loading the language you selected
